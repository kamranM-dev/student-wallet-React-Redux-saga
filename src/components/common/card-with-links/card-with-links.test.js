import React from 'react';
import Card from './card-with-links';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';

it('renders correctly with default params', () => {
  const links = [{ url: '#', text: 'link' }];
  const tree = shallow(
    <Card links={links} />
  );
  expect(toJson(tree)).toMatchSnapshot();
});
