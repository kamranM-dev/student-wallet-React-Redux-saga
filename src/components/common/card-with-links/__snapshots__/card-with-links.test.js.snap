exports[`test renders correctly with default params 1`] = `
<Card
  className="card-with-links">
  <a
    className="card-with-links__link"
    href="#"
    target="_blank">
    link
  </a>
</Card>
`;
